vendor/assets/pdf_forms/et3-v1-en.yml

```diff
@@ -98,57 +99,57 @@ earnings:
       true: 'yes'
     unselected_value: 'Off'
   correct_hours:
-    field_name: '4.1 if no'
+    field_name: 4.1 if no
   agree_with_earnings:
     field_name: '4.2'
     select_values:
       false: 'no'
       true: 'yes'
     unselected_value: 'Off'
   correct_pay_before_tax:
-    field_name: '4.2 pay before tax'
+    field_name: 4.2 pay before tax
   correct_pay_before_tax_period:
-    field_name: '4.2 pay before tax tick box'
+    field_name: 4.2 pay before tax tick box
     select_values:
-      Monthly: 'monthly'
-      Weekly: 'weekly'
+      Monthly: monthly
+      Weekly: weekly
     unselected_value: 'Off'
   correct_take_home_pay:
-    field_name: '4.2 normal take-home pay'
+    field_name: 4.2 normal take-home pay
   correct_take_home_pay_period:
-    field_name: '4.2 normal take-home pay tick box'
+    field_name: 4.2 normal take-home pay tick box
     select_values:
-      Monthly: 'monthly'
-      Weekly: 'weekly'
+      Monthly: monthly
+      Weekly: weekly
     unselected_value: 'Off'
   agree_with_notice_period:
-    field_name: '4.3 tick box'
+    field_name: 4.3 tick box
     select_values:
-      no: 'no'
-      yes: 'yes'
+      false: 'no'
```

Comment:
In general, the left side is what the test suite sees - I prefer true / false for yes/no fields as that is what is in my factories (as I am in the API not user facing)